tick/optim/model/tests/sccs_test.py

```diff
@@ -0,0 +1,148 @@
+import unittest
+import numpy as np
+from scipy.optimize import check_grad, fmin_bfgs
+from scipy.linalg import norm
+from scipy.sparse import csr_matrix
+from tick.optim.model.sccs import ModelSCCS
+from tick.simulation import SimuSCCS
+from tick.preprocessing import LongitudinalFeaturesLagger
+from tick.optim.solver import SVRG
+from tick.optim.prox import ProxZero
+
+
+class ModelSCCSTest(unittest.TestCase):
+
+    def setUp(self):
+        self.X = [np.array([[0, 1],
+                            [0, 1]], dtype="float64"),
+                  np.array([[1, 1],
+                            [1, 0]], dtype="float64")
+                 ]
+        self.y = [np.array([1, 0], dtype="int32"),
+                  np.array([0, 1], dtype="int32")
+                 ]
+
+    def test_loss(self):
+        """Test longitudinal multinomial model loss."""
+        # TODO: make this more complex
```

Comment:
Check what we do in other Models unitests if necessary. 
But it seems that's enough tests for now right ?